### PR TITLE
Updated hover effect & matched colors

### PR DESCRIPTION
**Pull request by ghost**
For this PR:

- I changed the opacity so that it doesn't decrease until you hover.
- I changed the green in the "accept mission" element to match the green in the HUD.

Trello: https://trello.com/c/ijA4FCW2

### CODE REVIEW

**Review by milklot — APPROVED** [review]
Nice job on a styling, good start with .less experience . Keep it up!

File: src/styles/less/ChildDashboard.less [hunk 1]
```diff
@@ -49,49 +47,49 @@
   }
 
   div.accept-mission {
-    border-width: .75rem .375rem .75rem .75rem;
+    border-width: 0.75rem 0.375rem 0.75rem 0.75rem;
```
Comment (rhiannonstanford) [inline src/styles/less/ChildDashboard.less:50]: Nice add on the zeros here.  I hadn't thought to do that.  Definitely makes the values more clear!

File: src/styles/less/ChildDashboard.less [hunk 2]
```diff
@@ -128,33 +126,31 @@
...
+  .single-btn {
     display: flex;
     justify-content: center;
     align-items: center;
     background: @bright-sun;
-    font-size:xx-large;
-    font-family: 'bangers',cursive;
-    margin-left:2.5%;
-    width:95%;
+    font-size: xx-large;
```
Comment (rhiannonstanford) [inline src/styles/less/ChildDashboard.less:150]: Good clean up here.  Nice work!

**Review by rhiannonstanford — APPROVED** [review]
Good job making the child dashboard styling congruent to the Figma and cleaning up the less syntax.

**Review by Veilios — APPROVED** [review]
Super clean and easy to read code. Awesome job!

**Pull request merged**